packages/@react-spectrum/progress/docs/ProgressCircle.mdx

```diff
@@ -0,0 +1,103 @@
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/progress';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {ProgressCircle} from '@react-spectrum/progress';
+import {View} from '@react-spectrum/view';
+```
+
+# ProgressCircle
+
+<p>{docs.exports.ProgressCircle.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ProgressCircle']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/circle-loader/'}
+  ]} />
+
+## Example
+
+```tsx example
+<ProgressCircle aria-label="Loading…" value={50} />
+```
+
+## Value
+
+By default Progress Circles are controlled with the `value` prop representing current percentage as an integer.
+In this case, the minimum and maximum values default to 0 and 100, respectively.
+
+```tsx example
+<ProgressCircle aria-label="Loading…" value={25} />
+```
+
+Alternatively, a different scale can be used by setting the `minValue` and `maxValue`.
+
+```tsx example
+<ProgressCircle aria-label="Loading…" minValue={50} maxValue={150} value={100} />
+```
+
+### Indeterminate
+[View guidelines](https://spectrum.adobe.com/page/circle-loader/#Indeterminate)
+
+By default, Progress Circles are determinate. Use a determinate Progress Circle when progress can be calculated against a specific goal.
+
+```tsx example
+<ProgressCircle aria-label="Loading…" value={50} />
+```
+
+Use an indeterminate Progress Circle when progress is happening but the time or effort to completion can’t be determined.
+
+```tsx example
+<ProgressCircle aria-label="Loading…" isIndeterminate />
+```
+
+### Accessibility
+
+ProgressCircle requires an `aria-label` or `aria-labelledby` attribute.
```

Comment:
This is very direct and clear, but I like `Meter's` way of saying this.
`Depending on the visualization being used (i.e. depending on the showValueLabel prop), a label, aria-label, or aria-labelledby attribute are required.`